apps/rush/src/actions/GenerateAction.ts

```diff
@@ -59,13 +58,13 @@ export default class GenerateAction extends CommandLineAction {
     const stopwatch: Stopwatch = Stopwatch.start();
     const isLazy: boolean = this._lazyParameter.value;
 
-    console.log('Starting "rush generate"' + os.EOL);
-
-    if (this._rushConfiguration.packageReviewFile) {
-      this._packageReviewChecker = new PackageReviewChecker(this._rushConfiguration);
-      this._packageReviewChecker.saveCurrentDependencies();
+    if (this._rushConfiguration.approvedPackagesPolicyEnabled) {
+      const packageReviewChecker: PackageReviewChecker = new PackageReviewChecker(this._rushConfiguration);
+      packageReviewChecker.saveCurrentDependencies();
     }
```

Comment:
does it make more sense to introduce a base ? I did this with evergreen as there was some common config and stopwatch stuff that I got tired of repeating